app/api/ada/adaTransactions/adaNewTransactions.js

```diff
@@ -46,107 +28,149 @@ const protocolMagic = CONFIG.network.protocolMagic;
 /** Calculate the transaction fee without actually sending the transaction */
 export async function getAdaTransactionFee(
   receiver: string,
-  amount: string
+  amount: string,
+  changeAdaAddr: ?AdaAddress,
+  possibleInputAddresses: Array<AdaAddress>,
```

Comment:
the possible naming is pretty good, maybe we should add a comment to explain why it’s like that

Reply:
you already added it :D

Reply:
I added more comments to explain what this means